### ISSUE TITLE
PyPi release outdated

### ISSUE THREAD

**Issue by labrys** [Diaoul/babelfish#33]
With Python 3.10 on the horizon and #29 implemented to fix the issue of importing `from collections` instead of `from collections.abc` PyPi could use a new release that reflects this fix.

**Comment by Safihre** [comment]
Could you please release the fix?
Thank you!

**Comment by miigotu** [comment]
@Diaoul I cannot push my package to pypi because of this:

```bash
 - Uploading sickchill-2021.7.12-py3-none-any.whl 100%

  HTTPError

  400 Client Error: Invalid value for requires_dist. Error: Can't have direct dependency: 'babelfish @ git+https://github.com/Diaoul/babelfish@master' for url: https://upload.pypi.org/legacy/

```

**Comment by miigotu** [comment]
I pushed up a temporary solution on pypi from master for now: https://pypi.org/project/temp-babelfish/
@Diaoul is there something holding it back from release?

**Comment by Diaoul** [comment]
Nothing but some time to do it. I'll get around it this week.
Sorry for the trouble.

**Comment by Diaoul** [comment]
0.6.0b1 is published to PyPI: https://pypi.org/project/babelfish/#history
Please let me know if this works for you, I did some basic testing on a clean install and it worked without issues.

**Comment by Safihre** [comment]
All our tests are confirming this is working as expected, a stable release would be nice :) @Diaoul

**Comment by miigotu** [comment]
I have yanked temp-babelfish, hopefully nobody was using it.

Fyi if anyone comes across this thread, they need to add `--pre` to their pip command or pin the version to get the pre-released version.

**Comment by Diaoul** [comment]
Just released 0.6.0 :tada:
